### PR TITLE
Use disclaimer to promote the Russian Apps 

### LINKED ISSUE #9396: Use semantic HTML for disclaimer component
**Is your feature request related to a problem? Please describe.**
The screenreader experience for the disclaimer component could be improved slightly by making the HTML more semantic.

**Describe the solution you'd like**
Update the Disclaimer component's HTML with these changes:

- Use a `section`, Landmark: Region with label ‘Disclaimer’ in the language of the service - e.g.`Уведомление` for Russian - defaults to English where translation isn't available for other services
- Use `strong` for the actual text

An example of how the rendered HTML should be presented:

```html
<section role="region" aria-label="Disclaimer">
  <strong>This is a disclaimer</strong>
</section>
```
**Accessibility Acceptance Criteria**
https://paper.dropbox.com/doc/Disclaimer-a11y-ac--BR4tmZLBuoP3o~3h3QpN85ffAg-kR9WZaIuJi73YyUIl3XKw

**Testing notes**
[Tester to complete]

Dev insight: Will Cypress tests be required or are unit tests sufficient? Will there be any potential regression? etc

- [x] This feature is expected to need manual testing.

**Checklist**
- [x] (BBC contributors only) This issue follows the [repository use guidelines](https://github.com/bbc/simorgh-infrastructure/blob/latest/documentation/repository-guidelines.md)

**Additional context**

### PR DESCRIPTION
**Pull request by DarioR01**
Resolves #9396 
Resolves https://github.com/bbc/simorgh-infrastructure/issues/1564

**Overall change:**
Added new disclaimer message with inLine links and uses semantic HTML.

We will have an a11y swarm for this component on at a second time due to its high priority.

**Code changes:**

- Added a new translation in Russian config for Section's aria-label (default to English 'Disclaimer')
- Added a new structure for the `disclaimer` in the Russian service's config.
- Convert the new disclaimer into an array of values and map through each value to allow dynamic inLine links

---

- [x] I have assigned myself to this PR and the corresponding issues
- [ ] I have added the `cross-team` label to this PR if it requires visibility across World Service teams
- [x] I have assigned this PR to the Simorgh project
- [ ] (BBC contributors only) This PR follows the [repository use guidelines](https://github.com/bbc/simorgh-infrastructure/blob/latest/documentation/repository-guidelines.md)

**Testing:**

- [x] Automated (jest and/or cypress) tests added (for new features) or updated (for existing features)
- [ ] If necessary, I have run the local E2E non-smoke tests relevant to my changes (`CYPRESS_APP_ENV=local CYPRESS_SMOKE=false yarn test:e2e:interactive`)
- [x] This PR requires manual testing

### CODE REVIEW

File: src/app/containers/Disclaimer/index.test.jsx [hunk 1]
```diff
@@ -26,7 +26,7 @@ const renderComponent = ({
   );
 
 describe('Disclaimer Component', () => {
-  it('Renders the text from the service config', () => {
+  it.skip('Renders the text from the service config', () => {
```
Comment (RetiredBlob) [inline src/app/containers/Disclaimer/index.test.jsx]: Could make this test work by changing line 22 from this:

```
<ServiceContext.Provider value={{ disclaimer: { text: disclaimerText } }}>
```

to something like this I think:

```
<ServiceContext.Provider value={{ disclaimer: { block: [{ text: disclaimerText }] } }}>
```
```suggestion
  it.skip('Renders the text from the service config', () => {
```
The test on line 34 is also useless if we don't make this change

File: src/app/containers/Disclaimer/index.jsx [hunk 2]
```diff
@@ -51,7 +52,18 @@ const DisclaimerComponent = ({ increasePaddingOnDesktop }) => {
...
+        {disclaimerBlock.map(element => {
+          const isLink = pathOr(false, ['link'], element);
+          const linkText = pathOr('', ['link', 'text'], element);
+          const linkHref = pathOr('', ['link', 'href'], element);
+          return isLink ? (
+            <InlineLink href={linkHref} key={linkText}>
+              {linkText}
+            </InlineLink>
+          ) : (
+            element?.text
```
Comment (RetiredBlob) [inline src/app/containers/Disclaimer/index.jsx]: Are there any console errors being output due to this not having a `key`?

**Review by RetiredBlob — APPROVED** [review]
👍

**Review by ryanmccombe — APPROVED** [review]

File: src/app/containers/Disclaimer/index.test.jsx (hunk 1 above)
Comment (RetiredBlob) [reply src/app/containers/Disclaimer/index.test.jsx]: Noticed this was fixed in the other PR

**Review by amoore108 — APPROVED** [review]

File: src/app/containers/Disclaimer/index.jsx (hunk 2 above)
Comment (DarioR01) [reply src/app/containers/Disclaimer/index.jsx]: Not that I can see :D, I believe react sees that as text only

File: src/app/containers/Disclaimer/index.test.jsx (hunk 1 above)
Comment (DarioR01) [reply src/app/containers/Disclaimer/index.test.jsx]: Testing is done on the other a11y-commit PR :D

File: src/app/containers/Disclaimer/index.test.jsx [hunk 3]
```diff
@@ -6,11 +6,26 @@ import { ToggleContextProvider } from '#contexts/ToggleContext';
 
 import DisclaimerComponent from '.';
 
+const DISCLAIMER_FIXTURE = {
+  para1: 'Приложение Русской службы BBC News доступно для ',
+  para2: {
+    text: 'IOS ',
+    url: 'https://apps.apple.com/us/app/%D0%BD%D0%BE%D0%B2%D0%BE%D1%81%D1%82%D0%B8-%D0%B1%D0%B8-%D0%B1%D0%B8-%D1%81%D0%B8/id504278066',
+  },
+  para3: 'и ',
```
Comment (ryanmccombe) [inline src/app/containers/Disclaimer/index.test.jsx]: Does this make the space after `IOS` also part of the link, and underlined?

Should maybe be this - find to merge without it though if urgent!

```suggestion
    text: 'IOS',
    url: 'https://apps.apple.com/us/app/%D0%BD%D0%BE%D0%B2%D0%BE%D1%81%D1%82%D0%B8-%D0%B1%D0%B8-%D0%B1%D0%B8-%D1%81%D0%B8/id504278066',
  },
  para3: ' и ',
```

**Pull request merged**